pkg/apis/monitoring/v1/types.go

```diff
@@ -742,6 +750,84 @@ type PodMetricsEndpoint struct {
 	ProxyURL *string `json:"proxyUrl,omitempty"`
 }
 
+// BlackboxMonitor defines monitoring for a set of static targets or ingresses.
+// +genclient
+// +k8s:openapi-gen=true
+type BlackboxMonitor struct {
+	metav1.TypeMeta `json:",inline"`
+	// Standard object’s metadata. More info:
+	// https://github.com/kubernetes/community/blob/master/contributors/devel/api-conventions.md#metadata
+	// +k8s:openapi-gen=false
```

Comment:
@auhlig  `+k8s:openapi-gen=false` does not seems to have any effects on native Kubernetes types. Thus, your comments generate a `description` field inside of the metadata schema which lead to a validation error.

Reply:
Yes. Thanks for pointing out. Not sure why it's not working though :-/